crates/storybook/src/stories/kitchen_sink.rs

```diff
@@ -9,6 +10,7 @@ pub struct KitchenSinkStory;
 
 impl KitchenSinkStory {
     pub fn view(cx: &mut WindowContext) -> View<Self> {
+        Project::init_settings(cx);
```

Comment:
I wonder if at this point we should just call `Project::init_settings` once on the main path (where we initialize the other relevant settings) instead of doing it in each individual story?

So we could add the call here: https://github.com/zed-industries/zed/blob/2ee257a5621d3c2714945e117d958383c14ae513/crates/storybook/src/storybook.rs#L81-L82

Reply:
@maxdeviant yes great idea !  I made the change you requested...  Thank you kindly for reviewing this PR...